culane_metric/evaluate.py

```diff
@@ -85,8 +85,24 @@ def continuous_cross_iou(xs, ys, width=30):
     return ious
 
 
+def remove_con_dup(x):
+    """Customize a set op to remove consecutive duplications"""
+    y = []
+    for t in x:
+        if len(y) > 0 and y[-1] == t:
```

Comment:
This should show a warning because it will change results for multiple duplicates. E.g., you remove 15 and suddenly the turn moved.